release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4234,3 +4234,35 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-13-44-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-45"]
+=== RHSA-2024:4484 - {product-title} 4.13.45 bug fix and security updates
+
+Issued: 2024-07-17
+
+{product-title} release 4.13.45, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4484[RHSA-2024:4484] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4486[RHSA-2024:4486] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
```

Comment:
Should this be commented out or removed?

Reply:
No, that's in every z-stream release note.